newm-chain/src/main/kotlin/io/newm/chain/grpc/NewmChainService.kt

```diff
@@ -326,4 +326,22 @@ class NewmChainService : NewmChainGrpcKt.NewmChainCoroutineImplBase() {
             throw e
         }
     }
+
+    override suspend fun snapshotNativeAssets(request: SnapshotNativeAssetsRequest): SnapshotNativeAssetsResponse {
+        try {
+            val snapshotEntries: List<SnapshotEntry> =
+                ledgerRepository.snapshotNativeAssets(request.policy, request.name).map { (stakeAddress, amount) ->
+                    snapshotEntry {
+                        this.stakeAddress = stakeAddress
+                        this.amount = amount
+                    }
+                }
+            return snapshotNativeAssetsResponse {
+                this.snapshotEntries.addAll(snapshotEntries)
+            }
+        } catch (e: Throwable) {
+            log.error("snapshotNativeAssets error!", e)
```

Comment:
Maybe use `log.error { }`

Reply:
I can't use `log.error { }` because it doesn't allow the exception information to propagate to Sentry.